src/bindings/c/tests/ov_test.cpp

```diff
@@ -40,14 +40,3 @@ std::map<ov_element_type_e, size_t> element_type_size_map = {{ov_element_type_e:
                                                              {ov_element_type_e::U16, 16},
                                                              {ov_element_type_e::U32, 32},
```

Comment:
Can we add tests to check that C element type size is equal to size of C++ the same element type?

Reply:
There is a map table for C element type and C++ element type, there should be equal from this perspective. 
https://github.com/openvinotoolkit/openvino/blob/8b781c93a9e0a7fa1b6c6fe15ba29eec4d7b964a/src/bindings/c/src/ov_tensor.cpp#L8

Maybe I didn't get your point, could you give more detail?